integration/benchnet2/flow/templates/access.yml

```
@@ -33,7 +33,20 @@ spec:
         nodeType: access
         service: flow
         networkId: {{ $.Values.networkId }}
-    spec:
+    spec:  
+      nodeSelector:
+        iam.gke.io/gke-metadata-server-enabled: "true"
+      serviceAccountName: "benchnet-configuration-reader"
+      initContainers:
+        - name: execution-download
```

```suggestion
        - name: bootstrap-download
```

Suggesting to change the name (also in the other `templates/*.yml` files for other roles), since we are downloading the whole bootstrap package, not just the execution snapshot.